src/Ombi/ClientApp/app/requests/tvrequests.component.ts

```diff
@@ -66,7 +66,19 @@ export class TvRequestsComponent implements OnInit {
 
     public openClosestTab(el: any) {
         const rowclass = "undefined ng-star-inserted";
-        el = el.toElement || el.relatedTarget || el.target;
+        el = el.toElement || el.relatedTarget || el.target || el.srcElement;
+
+        if (el.nodeName === "BUTTON") {
+
+            const isButtonAlreadyActive = el.parentElement.querySelector(".active");
+            // if a Button already has Class: .active
+            if (isButtonAlreadyActive) {
+                isButtonAlreadyActive.classList.remove("active");
+            } else {
+                el.className += " active";
+            }
+        }
+
```

Comment:
Similar idea to what is done above for movie requests component but instead we apply the active styling to the View button of the TV shows request page since current behavior is not correct in that when the button is toggled and clicked away it stays highlighted as if active.